[2024-05-21T14:44:07.604+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-21T14:44:07.640+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: statistical_calculations_dag.calculate_statistics scheduled__2024-01-19T00:00:00+00:00 [queued]>
[2024-05-21T14:44:07.654+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: statistical_calculations_dag.calculate_statistics scheduled__2024-01-19T00:00:00+00:00 [queued]>
[2024-05-21T14:44:07.656+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-05-21T14:44:07.681+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): calculate_statistics> on 2024-01-19 00:00:00+00:00
[2024-05-21T14:44:07.688+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=10749) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-21T14:44:07.690+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'statistical_calculations_dag', 'calculate_statistics', 'scheduled__2024-01-19T00:00:00+00:00', '--job-id', '4011', '--raw', '--subdir', 'DAGS_FOLDER/second_dag.py', '--cfg-path', '/tmp/tmpw8b5roor']
[2024-05-21T14:44:07.691+0000] {standard_task_runner.py:63} INFO - Started process 10760 to run task
[2024-05-21T14:44:07.693+0000] {standard_task_runner.py:91} INFO - Job 4011: Subtask calculate_statistics
[2024-05-21T14:44:07.757+0000] {task_command.py:426} INFO - Running <TaskInstance: statistical_calculations_dag.calculate_statistics scheduled__2024-01-19T00:00:00+00:00 [running]> on host c420abfdc210
[2024-05-21T14:44:07.906+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='test' AIRFLOW_CTX_DAG_ID='statistical_calculations_dag' AIRFLOW_CTX_TASK_ID='calculate_statistics' AIRFLOW_CTX_EXECUTION_DATE='2024-01-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-19T00:00:00+00:00'
[2024-05-21T14:44:07.908+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-21T14:44:07.980+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-21T14:44:07.981+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-21T14:44:07.993+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=statistical_calculations_dag, task_id=calculate_statistics, run_id=scheduled__2024-01-19T00:00:00+00:00, execution_date=20240119T000000, start_date=20240521T144407, end_date=20240521T144407
[2024-05-21T14:44:08.029+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-21T14:44:08.055+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-21T14:44:08.059+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
